bin/generate-podspecs.sh

```diff
@@ -196,8 +196,11 @@ do
         mv "$TMP_FBReactNativeSpec" "$DEST/FBReactNativeSpec/FBReactNativeSpec.podspec.json"
     fi
 done
+popd > /dev/null
 
-echo 'Updating XCFramework Podfile.lock with these changes'
-pushd ios-xcframework
-bundle exec pod update
-popd
+if [[ "$COMMIT_HASH" != "0" ]]; then
```

Comment:
A someone with little knowledge of this setup, it took some digging to understand where the 0 value came from.


```suggestion
# We are required to run this script twice to capture the correct target.
# 0 is the value set on the first running this script to generate the podspecs.
if [[ "$COMMIT_HASH" != "0" ]]; then
```

Reply:
Adding a note makes total sense. Thank you.